### DISCUSSION TITLE
Why the wallet icon always twinkle

### DISCUSSION THREAD

**Discussion by ghost** [WalletConnect/walletconnect-monorepo#3972] [Developer Support] [answered]
<img width="371" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/133619503/43e62ed5-0068-48ab-bd52-1f096f84c5bc">
When I  open the connect panel. These wallet icons always twinkle. I don't know why? Anyone can help me>

**Comment by glitch-txs** [comment, marked as answer]
hi, this is not our SDK, I'd recommend integrating Web3Modal instead and we can assist on any questions related.

https://docs.walletconnect.com/web3modal/about
